Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM debian:jessie
```

Comment:
Why the old Debian version?

Reply:
Feel free to update it in a separate PR :slightly_smiling_face:

Reply:
See https://github.com/coredump-ch/status/pull/123